--- /usr/local/lib/python3.8/dist-packages/transformers/modeling_utils.py	2023-08-04 20:31:13.000000000 +0000
+++ checkpoints/modeling_utils.py	2023-08-05 14:58:30.370824590 +0000
@@ -2999,11 +2999,11 @@
         if len(cls.base_model_prefix) > 0 and hasattr(model, cls.base_model_prefix) and not has_prefix_module:
             model_to_load = getattr(model, cls.base_model_prefix)
             base_model_expected_keys = list(model_to_load.state_dict().keys())
-            if any(key in expected_keys_not_prefixed and key not in base_model_expected_keys for key in loaded_keys):
-                raise ValueError(
-                    "The state dictionary of the model you are trying to load is corrupted. Are you sure it was "
-                    "properly saved?"
-                )
+            #if any(key in expected_keys_not_prefixed and key not in base_model_expected_keys for key in loaded_keys):
+            #    raise ValueError(
+            #        "The state dictionary of the model you are trying to load is corrupted. Are you sure it was "
+            #        "properly saved?"
+            #    )
             if device_map is not None:
                 device_map = {k.replace(f"{cls.base_model_prefix}.", ""): v for k, v in device_map.items()}
 
